src/components/EditUserMetadata.jsx

```diff
@@ -46,7 +47,11 @@ export default function EditUserMetadata({
 
   useEffect(
     () => {
-      setFieldValues(getInitialFormValues(metadata));
+      const safeMetadata = filter(
+        metadata,
+        datum => Object.keys(datum).length > 1,
```

Comment:
the malformed twitter stand-in metadata element had only one key called value. This seems like one way to remove it.